tethys_gizmos/gizmo_options/plot_view.py

```diff
@@ -1,39 +1,15 @@
 # coding=utf-8
 from .base import TethysGizmoOptions
 
```

Comment:
I took it out as it would conflict with the gizmo_name of the new plot view. Maybe should throw an error that leads them to the new gizmo instructions.